lib/languages/japanese.sty

```diff
@@ -58,13 +59,13 @@
     % Innate spellcasting
     \renewcommand\innateatwillname{無限回}
     \renewcommand\numberperdayname{|1|回／日}
-    \renewcommand\numberperdayeachname{|1|回／日ずつ} % TODO: Confirm official translation
+    \renewcommand\numberperdayeachname{各|1|回／日}
     % Spell header
     \renewcommand\spellcastingtimename{発動時間}
     \renewcommand\spellrangename{射程}
     \renewcommand\spellcomponentsname{構成要素}
     \renewcommand\spelldurationname{持続時間}
     % Miscellaneous
+    \renewcommand\pageabbreviationname{pg.}
     \renewcommand\dname{d}
-%    \newcommand\tocchapterabbreviationname{Ch.}
```

Comment:
How about putting in as much of the translation as you can, even if the order of the words is wrong. Otherwise, you are going to get "Ch." in the table of contents. If the English is preferable to Japanese words out of order, still please replace the commented command and place the correct parameterized version at the end of the comment. When we add parameters to this and "pg.", we can then automatically update the text without you needing to do another update down the road.

Reply:
Thanks for the review.
I added translations for `\tocchapterabbreviationname` and `\pageabbreviationname`, with a comment to update them when #308 is closed.
I also noticed that `\tocchapterabbreviationname` was missing from `_template.sty`, so I added it.